src/java/org/jivesoftware/openfire/plugin/util/cache/ClusteredCache.java

```diff
@@ -148,9 +148,9 @@ public long getCacheMisses() {
     }
 
     @Override
-    public int getCacheSize() {
+    public long getCacheSize() {
         final LocalMapStats stats = map.getLocalMapStats();
-        return (int) (stats.getOwnedEntryMemoryCost() + stats.getBackupEntryMemoryCost());
+        return stats.getOwnedEntryMemoryCost() + stats.getBackupEntryMemoryCost();
```

Comment:
Not something you've done, but something highlighted by the change - if the Openfire cache size is bytes, and Hazelcast cache size is Megabytes, does this need multiplying up?

Reply:
I think this is not a problem, because the java doc says both methods return the size in bytes:

```
/**
     * Returns memory cost (number of bytes) of owned entries in this member.
     *
     * @return memory cost (number of bytes) of owned entries in this member.
     */
    long getOwnedEntryMemoryCost();

    /**
     * Returns memory cost (number of bytes) of backup entries in this member.
     *
     * @return memory cost (number of bytes) of backup entries in this member.
     */
    long getBackupEntryMemoryCost();
```